inst/README.markdown

```diff
@@ -197,7 +197,9 @@ will allow successful installation of these packages.
   installation for warning messages. Warnings usually mean installation
   errors, so by default remotes stops for a warning. However, sometimes
   other warnings might happen, that could be ignored by setting this
-  environment variable.
+  environment variable. Warnings are also allowed if
+  `R_REMOTES_NO_ERRORS_FROM_WARNINGS` is unset and `_R_CHECK_FORCE_SUGGESTS_`
+  is `FALSE`.
```

Comment:
I think this is confusing. This paragraph is about suppressing the  warning -> error conversion. So we don't need to modify it. Maybe have a new item for `_R_CHECK_FORCE_SUGGESTS_`?